src/User32.Desktop/User32.cs

```diff
@@ -68,6 +69,16 @@ public static partial class User32
         [DllImport(nameof(User32), SetLastError = true)]
         public static extern int GetWindowLong(IntPtr hWnd, WindowLongIndexFlags nIndex);
 
+        [DllImport(nameof(User32), SetLastError = true, CharSet = CharSet.Auto)]
+        public static extern int GetClassName(IntPtr hWnd, StringBuilder lpClassName, int nMaxCount);
+
+        [DllImport(nameof(User32), SetLastError = true)]
+        public static extern uint GetWindowThreadProcessId(IntPtr hWnd, out uint lpdwProcessId);
```

Comment:
Yeah, makes sense. I see it on https://msdn.microsoft.com/en-us/library/system.diagnostics.processthread.id(v=vs.110).aspx and https://msdn.microsoft.com/en-us/library/system.diagnostics.process.id(v=vs.110).aspx

That gives us the possibility of negative process and thread IDs, wich is weird though. BUt I agree that if the .NET platform considers this as int, so should we.